tasks/analysis.py

```diff
@@ -296,6 +297,16 @@ def process_impact_result(self, impact_result, analysis_id):
     if impact_result['status'] == RESULT_SUCCESS:
         impact_url = impact_result['output']['impact_analysis']
 
+        # generate report when analysis has ran successfully
+        async = generate_report.delay(impact_url)
+        with allow_join_result():
+            report_metadata = (
+                async.get().get('output', {}).get('pdf_product_tag', {}))
```

Comment:
@lucernae I don't know if this is the correct way to call the task and get the result from it. It works but it takes time to fetch the result after the `generate_report` task finished. Please let me know if there is a better way of doing it.